### PR TITLE
Migrate counter and gauge docs to new format

### PR DESCRIPTION
**Pull request by charislam**
# Description

[Short summary of why you created this PR]

# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: api/_hyperfunctions/gauge_agg/gauge_agg.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  type: aggregate
+  aggregates:
+    - gauge_agg()
+api_details:
+  summary: |
+    This is the first step for performing any aggregate calculations
+    on gauge data. Use `gauge_agg` to create an intermediate aggregate
+    from your data. This intermediate form can then be used
+    by one or more accessors in this group to compute final results. Optionally,
+    multiple such intermediate aggregate objects can be combined using
```
Comment (Loquacity) [inline api/_hyperfunctions/gauge_agg/gauge_agg.md:23]:
```suggestion
    you can combine multiple intermediate aggregate objects with
```

File: api/_hyperfunctions/gauge_agg/gauge_zero_time.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  version:
+    experimental: 1.6.0
+hyperfunction:
+  family: counters and gauges
+  type: accessor
+  aggregates:
+    - gauge_agg()
+api_details:
+  summary: >
+    Calculate the time when the gauge value is predicted to have been zero.
```
Comment (Loquacity) [inline api/_hyperfunctions/gauge_agg/gauge_zero_time.md:19]: 'predicted' feels like an event that happens in the future, but this is working out something from the past, right? Maybe "calculated"?

**Review by Loquacity — APPROVED** [review]
Some minor suggestions, nothing blocking.

File: api/_hyperfunctions/gauge_agg/gauge_zero_time.md (hunk 2 above)
Comment (charislam) [reply api/_hyperfunctions/gauge_agg/gauge_zero_time.md:19]:
```suggestion
    Calculate the time when the gauge value is modeled to have been zero.
```

**Pull request merged**